mod-source-record-storage-server/src/main/resources/templates/db_scripts/create_get_record_by_instance_id_function.sql

```diff
@@ -0,0 +1,26 @@
+-- Custom script to create a function to get the MARC record by instanceId. Changes in this file will not result in an update of the function.
+-- To change the function, update this script and copy it to the appropriate scripts.snippet field of the schema.json
+
+CREATE OR REPLACE FUNCTION get_record_by_instance_id(instanceId uuid)
+RETURNS jsonb AS $rcrd$
+DECLARE
+	rcrd jsonb;
+BEGIN
+WITH rslt AS (SELECT *
```

Comment:
Is it possible to use result instead of rslt?

Reply:
as for the script - I will make a note in https://issues.folio.org/browse/MODSOURCE-62 and address it there, because any manipulation with the script requires thorough testing and time is scarce at the time